src/Forms/UWP/ArcGISRuntime.Xamarin.Forms.UWP.csproj

```diff
@@ -19,7 +19,7 @@
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <PackageCertificateKeyFile>Windows_TemporaryKey.pfx</PackageCertificateKeyFile>
-    <RuntimeIdentifiers>win10-arm;win10-arm-aot;win10-x86;win10-x86-aot;win10-x64;win10-x64-aot</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win10-arm;win10-arm-aot;win10-arm64-aot;win10-x86;win10-x86-aot;win10-x64;win10-x64-aot</RuntimeIdentifiers>
```

Comment:
I see `win10-arm64-aot` but how about `win10-arm64`?  https://docs.microsoft.com/en-us/dotnet/core/rid-catalog#windows-rids
To be perfectly honest I'm not sure if both are needed, but I bet @dotMorten knows :)